ddtrace/contrib/kafka/patch.py

```diff
@@ -23,7 +25,9 @@
 
 config._add(
     "kafka",
-    dict(_default_service="kafka"),
+    dict(
+        _default_service=schematize_service_name("kafka"),
```

Comment:
Can move this logic to IntegrationConfig.service. Again I am worried about maintainbility. We should try to make this change in the fewest places possible or we can end up some integrations with inconsistent service naming.

Reply:
This will work for integrations that use the `ext_service` function, but only a subset of integrations use that, or have this variable defined. For instance, HTTP Servers use int_service and several DB tie-ins (redis, mariadb, etc) use Pin overrides on top of the dbapi module?

If I were to take this course of action, the `config.py:Config` code would look something like:

```
def _add(self, integration, settings, merge=True):
    existing = getattr(self, integration)
    settings = deepcopy(settings)
    for k, v in settings:
         if k == "_default_service":
            settings[k] = schematize_service_name(v)
```

Reply:
Yup, I believe we have a long term goal of consolidating these mechanisms. Ideally there should be one api that can create and configuration a span for an integration. We can defer this work for now.